lib/pub_sub/active_record/callbacks.rb

```diff
@@ -90,16 +90,30 @@ def broadcast_after_destroy
           def broadcast_after_create_commit
             broadcast_callback_event(Events::AfterCommitEvent)
             broadcast_callback_event(Events::AfterCreateCommitEvent)
+            broadcast(Events::CreateChangeEvent.new({
+              timestamp: self.try(:updated_at) || Time.now(),
```

Comment:
Can we just use `Time.current`? This is the event timestamp, so it does feel more technically "true" to just use the current time we are broadcasting it.

Any reason they would have to exactly match with the AR record?

Reply:
If we ever replay them. We want to replay them in the order the DB applied them. This could, in theory, be an issue when multiple threads are at play.

Reply:
Hmmm. The order shouldn't matter but sure, this would make the replay more "pure".